src/components/ShareStructure/ShareStructure.vue

```diff
@@ -0,0 +1,1058 @@
+                          <v-icon>mdi-arrow-down</v-icon> Move Down
+                        </v-list-item-subtitle>
+                      </v-list-item>
+                      <v-list-item class="actions-dropdown_item" @click="removeSeries(index, row.index)">
+                        <v-list-item-subtitle>
+                          <v-icon>mdi-delete</v-icon> Remove
+                        </v-list-item-subtitle>
+                      </v-list-item>
+                    </v-list>
+                  </v-menu>
+                </span>
+              </div>
+            </td>
+          </tr>
+          <!-- Series Share Edit Form -->
+          <tr
+            v-if="showSeriesEditForm[row.index] && showSeriesEditForm[row.index][index]"
+            :key="`class:${row.index}-Series:${index}-edit-form`"
+          >
+            <td colspan="6">
+              <v-expand-transition>
+                <div class="edit-share-structure-container">
+                  <edit-share-structure
+                    :initialValue="currentShareStructure"
+                    :activeIndex="activeIndex"
+                    :shareId="shareId"
+                    :parentIndex="parentIndex"
+                    :shareClasses="shareClasses"
+                    :resolution-required="resolutionRequired"
+                    @addEditClass="addEditShareClass($event)"
+                    @addEditSeries="addEditShareSeries($event)"
+                    @removeSeries="removeSeries($event, row.index)"
+                    @resolutionPrompt="emitResolutionPrompt($event)"
+                    @resetEvent="resetData()"/>
+                </div>
+              </v-expand-transition>
+            </td>
+          </tr>
+        </template>
+
+        <!-- Series Share Add Form -->
+        <tr v-if="showSeriesAddForm[row.index]">
+          <td colspan="6">
+            <v-expand-transition>
+              <div class="edit-share-structure-container">
+                <edit-share-structure
+                  :initialValue="currentShareStructure"
+                  :activeIndex="activeIndex"
+                  :shareId="shareId"
+                  :parentIndex="parentIndex"
+                  :shareClasses="shareClasses"
+                  :resolution-required="resolutionRequired"
+                  @addEditClass="addEditShareClass($event)"
+                  @addEditSeries="addEditShareSeries($event)"
+                  @removeSeries="removeSeries($event, row.index)"
+                  @resolutionPrompt="emitResolutionPrompt($event)"
+                  @resetEvent="resetData()"/>
+              </div>
+            </v-expand-transition>
+          </td>
+        </tr>
+      </template>
+    </v-data-table>
+  </v-card>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Emit, Mixins, Prop, Vue, Watch } from 'vue-property-decorator'
+import 'array.prototype.move'
+import { cloneDeep, isEqual, omit } from 'lodash'
```

Comment:
Should add dependency on lodash to package.json.

Reply:
Good catch, will do

Reply:
I'm ** hoping ** that won't cause another (nested) copy of lodash in the node_modules folder...

Reply:
If it does... Maybe we abandon the "self sufficient shared component" idea and just rely on the app's dependencies -- but add a comment to that effect in the code here.

Alternately, if we must have self-sufficient components then you could import just the lodash methods you need (eg, https://www.npmjs.com/package/lodash.clonedeep etc) instead of the whole library.

Reply:
> just rely on the app's dependencies

A slippery slope. How much should this component add to its package.json and how much should it depend on the app? All? None? I don't have an answer but we should try to answer this.

Reply:
Per chat in RC, we can determine this per scenario.

Reply:
> I'm ** hoping ** that won't cause another (nested) copy of lodash in the node_modules folder...

Does it? 😬

Reply:
I'm not sure how to tell. 
By including it here, it will include it in the Node_Modules for the Shared Components. 
Is that the one you are referring too?

Reply:
I'm hoping there aren't multiple, full copies of lodash...
node_modules/lodash
node_modules/bcrs-shared-components/your_component/lodash
and others...

Reply:
ah i see. 
I suspect we will have..

node_modules/lodash
node_modules/bcrs-shared-components/your_component/lodash

Thoughts on how to resolve this?

Reply:
I'm not sure how to do this.. 

Is there a way to make it a dependency of the root app, if they pull in this component IF they don't already have it?

Reply:
Or we just pull in the specific functions in the shared comp.
 I believe you suggested that idea already.

Reply:
> Is there a way to make it a dependency of the root app, if they pull in this component IF they don't already have Lodash?

I don't think so. But the developer pulling it in will surely see the broken build.

Argh, I don't know which I prefer (or hate the least)... If we depend on the parent then what if they provide an incompatible version of the import? (bad parenting, haha) Dunno. Keep it in mind, decide later.